installation/scripts/testing-common.sh

```diff
@@ -74,10 +94,12 @@ function getContainerFromPod() {
     local namespace="$1"
     local pod="$2"
     local containers2ignore="istio-init istio-proxy manager"
-    containersInPod=$(kubectl get pods ${pod} -o jsonpath='{.spec.containers[*].name}' -n ${namespace})
-    for container in $containersInPod; do
-        if [[ ! ${containers2ignore[*]} =~ "${container}" ]]; then
-            echo "${container}"
+
+    result=$(executeKubectlWithRetries "kubectl get pods ${pod} -o jsonpath={.spec.containers[*].name} -n ${namespace}")
```

Comment:
how are u asserting the return code?

right now in err case, u will also iterate over err msg.

same problem for other executions